drivers/storage/virtualbox/virtualbox.go

```diff
@@ -0,0 +1,625 @@
+package virtualbox
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/akutz/gofig"
+	"github.com/akutz/goof"
+	"github.com/appropriate/go-virtualboxclient/virtualboxclient"
+
+	"github.com/emccode/rexray/core"
+	"github.com/emccode/rexray/core/errors"
+)
+
+const providerName = "VirtualBox"
```

Comment:
Hi @clintonskitson,

All the provider names get lcased anyway, so I'm not sure if we want to continue to describe them in the constants with mixed-case as later comparisons may be in err. Also, may I suggest `vbox` for brevity? I think they use that internally.